src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -39,19 +36,29 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
 
 /**
  * Used to set saturation of players. Number is used in case something changes in future...
  * @author bensku
+ * 
+ * Cleaned up by LimeGlass (2.2-dev35)
  */
 @Name("Saturation")
-@Description("The saturation of a player.")
-@Examples({"saturation of player is 20 #Not hungry!"})
-@Since("2.2-Fixes-V10")
+@Description("The saturation of the player(s).")
+@Examples("set saturation of player to 20 #Full hunger")
```

Comment:
Full hunger? It does not seem right...

Reply:
Each hunger is considered 1/20 or 0.5/10

20 is full hunger.

Reply:
I think @bensku talks about the sentence, the current sentence would mean hunger = 0.

Reply:
A hunger is not something that can be full. It cannot be measured. Thus, full hunger makes no sense grammatically.

Reply:
Well, a couple of things:

1. In the context of Minecraft, I'd personally say you can definitely consider hunger "full" since we very specifically have a bar measuring hunger which can be filled. Maybe "full hunger bar" would make everyone happy, but I think both are correct.
2. This is wrong in the first place since this is saturation, not hunger. Changing the player's saturation doesn't change their hunger, they're two separate bars. Saturation is basically a hunger shield (first you lose saturation, then hunger).
3. According to the wiki, your current max saturation isn't even always 20 -- its max value is [the same as your current hunger level](https://minecraft.gamepedia.com/Hunger#Mechanics), which would mean saturation can only be 20 if your hunger bar is full.

IMO the `#Full hunger` comment should be removed (since this expression is straight up not for hunger) and optionally be replaced with `# Max saturation`.